services/py-api/src/database/repository/teams_repository.py

```diff
@@ -26,21 +26,27 @@ async def create(
         self,
         input_data: ParticipantRequestBody,
         session: Optional[AsyncIOMotorClientSession] = None,
-        **kwargs: Dict[str, Any]
+        **kwargs: Dict[str, Any],
     ) -> Result[Team, DuplicateTeamNameError | Exception]:
 
+        session_id = session.session_id["id"].hex() if session is not None else None
```

Comment:
```python
session_id = session.session_id["id"].hex() if session else None
```

Reply:
ok

Reply:
Fixed in 62ecace76fd209bffa772b0238adedf16bd38dc2